unpackaged/config/dev/profiles/Admin.profile

```diff
@@ -75,7 +79,11 @@
         <visibility>DefaultOn</visibility>
     </tabVisibilities>
     <tabVisibilities>
-        <tab>%%%NAMESPACE%%%Time_Block__c</tab>
```

Comment:
Was this intentionally removed?

Reply:
looks like an accident

Reply:
This is some duplicate code I am trying to clean up.

Reply:
Ah, I see now. Perfect!